build.githubactions.ps1

```diff
@@ -232,30 +232,35 @@ function Get-IsWindows {
 }
 
 function InstallCredentialHandler {
-    if (Get-IsWindows -and -not Get-InstalledModule | Where-Object -Property Name -eq "7Zip4Powershell") {
-         Install-Module -Force -Scope CurrentUser -Name 7Zip4Powershell
+    if (Get-IsWindows)  {
+        $moduleNames = Get-Module -ListAvailable | Select-Object -ExpandProperty Name
+        if ($moduleNames -contains '7Zip4Powershell') {
+            Write-Host "'7Zip4Powershell' exists in available modules."
+        } else {
+            Install-Module -Force -Scope CurrentUser -Name 7Zip4Powershell
+            Write-Host "Installed 7Zip4Powershell."
+        }
+    }
```

Comment:
Can we revert the changes made to this condition? We have this condition in many of our PowerShell scripts, so it should work as it was.
Did you get an error?

Reply:
Yes I am geting error

Reply:
I fixed it . Thanks